airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -388,16 +407,17 @@ def _es_read(self, log_id: str, offset: int | str) -> ElasticSearchResponse | No
             }
         }
 
+        index_patterns = self._get_index_patterns(ti)
```

Comment:
Is index_patterns not required as class attribute?

Reply:
No, it should not